prometheus/prometheus.py

```diff
@@ -97,72 +97,79 @@ def collect(self):
             'How many HTLCs are currently active on this channel?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
 
         # Incoming routing statistics
         in_payments_offered_gauge = GaugeMetricFamily(
             'lightning_channel_in_payments_offered',
             'How many incoming payments did we try to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         in_payments_fulfilled_gauge = GaugeMetricFamily(
             'lightning_channel_in_payments_fulfilled',
             'How many incoming payments did we succeed to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         in_msatoshi_offered_gauge = GaugeMetricFamily(
             'lightning_channel_in_msatoshi_offered',
             'How many incoming msats did we try to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         in_msatoshi_fulfilled_gauge = GaugeMetricFamily(
             'lightning_channel_in_msatoshi_fulfilled',
             'How many incoming msats did we succeed to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
 
         # Outgoing routing statistics
         out_payments_offered_gauge = GaugeMetricFamily(
             'lightning_channel_out_payments_offered',
             'How many outgoing payments did we try to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         out_payments_fulfilled_gauge = GaugeMetricFamily(
             'lightning_channel_out_payments_fulfilled',
             'How many outgoing payments did we succeed to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         out_msatoshi_offered_gauge = GaugeMetricFamily(
             'lightning_channel_out_msatoshi_offered',
             'How many outgoing msats did we try to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
         out_msatoshi_fulfilled_gauge = GaugeMetricFamily(
             'lightning_channel_out_msatoshi_fulfilled',
             'How many outgoing msats did we succeed to forward?',
-            labels=['id', 'scid'],
+            labels=['id', 'scid', 'alias'],
         )
 
         peers = self.rpc.listpeers()['peers']
         for p in peers:
             for c in p['channels']:
-                labels = [p['id'], c.get('short_channel_id', c.get('channel_id'))]
+	        # append alias for human readable labels, if no label is found fill with shortid.
+                node = self.rpc.listnodes(p['id'])['nodes']
+                if len(node) != 0 and 'alias' in node[0]:
+                    alias = node[0]['alias']
+                else:
+                    alias = c.get('short_channel_id', c.get('channel_id'))
```

Comment:
Hmm we already have the scid, and (personal flavour here) i don't find the channel_id to be an useful user interface.
Could we just explicit it's unknown in this case ?
```python
alias = node[0].get("alias", "unknown")
```

Reply:
Problem is that switching from `short_channel_id` to the alias will cause a single timeseries to show up as two different series in the plot, which is quite confusing. I'd prefer just using a placeholder or leaving it out completely until we know the alias. For reference: we won't know the alias until our peer has a) opened and confirmed a channel with at least 6 confirmations and b) sent us a `node_announcement`.